modules/Emails/Email.php

```diff
@@ -1278,7 +1278,7 @@ public function email2Send($request)
                     if (!empty($note->id)) {
                         $filename = $note->filename;
                         $noteGUID = preg_replace('/[^a-z0-9\-]/', "", $note->id);
-                        $fileLocation = "upload://{$noteGUID}";
+                        $fileLocation = "upload/{$noteGUID}";
```

Comment:
Hi @Dillon-Brown,

Why do we need this change?

Reply:
It's for compatibility with PHPMailer 6.4.x. I've pulled in the change from: https://github.com/salesagility/SuiteCRM/pull/6938. Which was to fix https://github.com/salesagility/SuiteCRM/issues/6932 when we upgraded to 6.4.x in 7.11.x.

Reply:
ok, thanks Dillon